### ISSUE TITLE
Incorrect typings for counter controls on useCountdown

### ISSUE THREAD

**Issue by michaellopez** [EricLambrecht/react-timing-hooks#40]
**Bug Description:**
Currently, the types for the return value of `useCountdown` says it returns `[number, IntervalControls]` (https://github.com/EricLambrecht/react-timing-hooks/blob/b1c8ee9a15cc48fa00375e4c9a528e7ec2c0838a/src/interval/useCountdown.ts#L31C4-L31C30) when in fact it is using `useCounter` under the hood, therefore should return `CounterControls` from `useCounter`. The `reset` function is there, but right now TS throws error because the types are incorrect.

**To Reproduce:**
```
const [counter, counterControls] = useCountdown(10, 0, {
    onEnd: () => console.log("BOOM!"),
  });
  console.log({ counterControls });
  counterControls.reset() // TS gives error here, should not
```

**Expected behavior:**
counterControls.reset() should not give error in TS

**Screenshots:**

![image](https://github.com/user-attachments/assets/ddc1656f-180c-4edb-93ae-0a1d5701a6b7)
<img width="1005" alt="image" src="https://github.com/user-attachments/assets/f109c45e-fcda-45ed-9a4c-93d593d6f9b3">

**System:**
- Node Version: [e.g. 18.19.0 LTS]
- OS: macOS

**Additional context:**
N/A

**Comment by EricLambrecht** [comment]
@michaellopez Thank you for submitting a bug ticket. The bug has now been fixed in 5.0.2.

**Comment by michaellopez** [comment]
Thanks for the quick fix, much appreciated
